tests/test_authentication.py

```
@@ -32,40 +33,111 @@ def tearDown(self):
+                    pass
+
+            mock_session_local.return_value = DummySession()
+
+            response = self.client.post(
+                "/login", data={"username": USER_NAME, "password": "wrong_password"}
+            )
+            self.assertEqual(response.status_code, 200)
 
     def test_login_failure_lockout(self):
         # Simulate multiple failed login attempts to trigger lockout
         login_attempts = {} # reset 
```

Ruff (0.11.9)

Redefinition of unused `login_attempts` from line 16

(F811)